docs/MANUAL-TESTING.md

```diff
@@ -0,0 +1,89 @@
+Manual Testing
+==============
+
+This document describes a high-level script of manual tests to check for. We
+should aim to replace items on this list with automated Spectron test cases.
+
+Image Selection
+---------------
+
+- [ ] Cancel image selection dialog
+- [ ] Select an unbootable image (without a partition table), and expect a
+  sensible warning
+- [ ] Attempt to select an unsupported image
+- [ ] Change image selection
+
+Drive Selection
+---------------
+
+- [ ] Open the drive selection modal
+- [ ] Remove all drives
+- [ ] Switch drive selection
```

Comment:
Should we also check that the drive-selection list displays warnings for drives that are too small for the selected image?

Reply:
Makes sense!

Reply:
In fact, probably makes sense to check all the different warnings the drive-selector can display (system drive, drive too small, locked drive, etc.)

Reply:
Added!